README.md

```diff
@@ -0,0 +1,138 @@
+# New Document# **ALTERNATE BIT PROTOCOL SIMULATOR IMPLEMENTED IN CADMIUM**

```

Comment:
New Document is not needed before the title

Reply:
removed.